.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# Default ignored files
+/workspace.xml
```

Comment:
This should be in your `$HOME/.gitignore`

Reply:
ups!

Reply:
ignoring the `.idea` folder in `$HOME/.gitignore` and removing it from the repo.